package.json

```diff
@@ -1,28 +1,29 @@
 {
-    "name": "mediainfo-wrapper",
-    "version": "1.2.0",
-    "description": "wrapper for mediainfo-prebuilt",
-    "main": "index.js",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/vankasteelj/mediainfo-wrapper/issues"
-    },
-    "dependencies": {
-        "xml2js": "^0.4.16",
-        "glob": "^7.0.3"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/vankasteelj/mediainfo-wrapper.git"
-    },
-    "keywords": [
-        "mediainfo",
-        "media",
-        "metadata",
-        "codecs"
-    ],
-    "author": {
-        "name": "Jean van Kasteel",
-        "email": "vankasteelj@gmail.com"
-    }
+  "name": "mediainfo-wrapper",
+  "version": "1.3.1",
```

Comment:
From 1.2.0, you need to update to 2.0.0 because it's incompatible with v1 (options in first object change, so ppl need to update their code = breaking change, see http://semver.org/)